corehq/apps/case_search/management/commands/compare_case_properties.py

```diff
@@ -0,0 +1,146 @@
+from django.core.management.base import BaseCommand, CommandError
+from corehq.apps.es.case_search import CaseSearchES
+import timeit
+import json
+
+
+# Returns min, max, average of test suite results list
+def get_stats(results_list):
+    n = len(results_list)
+    if n == 1:
+        return results_list[0], results_list[0], results_list[0]
+    min_value = min(results_list)
+    max_value = max(results_list)
+    avg_value = sum(results_list) / n
+    return min_value, max_value, avg_value
+
+
+def run_baseline_query(query_obj):
+    print("Running baseline query...")
+    result = query_obj.run().total
+    print("Done!")
+    return result
+
+
+def run_test_queries(query_obj, repeats):
+    results = []
+    hits = query_obj.run().total
+    print("Running test queries...")
+    for _ in range(repeats):
+        results.append(timeit.timeit(query_obj.run, number=1))
```

Comment:
since we want to record the `took` value from each query I think it will be simpler to use `time.perf_counter()`:
```python
timings = []
for _ in range(repeats):
    start = time.perf_counter()
    result = query_obj.run()
    took = result.raw['took']
    end = time.perf_counter()
    diff = end - start  # duration in seconds
    # use a tuple to store both results
    # You could use a namedtuple to make accessing the values easier: https://docs.python.org/3/library/collections.html#collections.namedtuple
    timings.append((diff, took))
```

Reply:
e2728f188585167d55868939c5688ca1f2c82928